packages/api/src/store/table.ts

```diff
@@ -0,0 +1,183 @@
+import sql, { SQLStatement } from 'sql-template-strings'
+import DB from './db'
+import logger from '../logger'
+import { BadRequestError, NotFoundError } from './errors'
+
+interface TableSchema {
+  table: string
+}
+
+interface DBObject {
+  id: string
+}
+
+interface FindQuery {
+  [key: string]: any
+}
+
+interface FindOptions {
+  cursor?: string
+  limit?: number
+}
+
+export default class Table<T extends DBObject> {
+  db: DB
+  schema: TableSchema
+  name: string
+  constructor({ db, schema }) {
+    this.db = db
+    this.schema = schema
+    this.name = schema.table
+  }
+
+  // get a single document by id
+  async get(id: string): Promise<T> {
+    if (!id) {
+      throw new Error('missing id')
+    }
+    const res = await this.db.query(
+      sql`SELECT data FROM `.append(this.name).append(sql` WHERE id=${id}`),
```

Comment:
```suggestion
      sql`SELECT data FROM `.append(this.name).append(sql` WHERE id=${id}`.setName(`${this.name}_by_id`)),
```
to use Postres prepared statement